astropy/visualization/wcsaxes/tests/test_display_world_coordinates.py

```diff
@@ -31,7 +31,7 @@ def test_overlay_coords(self, ignore_matplotlibrc, tmp_path):
 
         # Testing default displayed world coordinates
         string_world = ax._display_world_coords(0.523412, 0.518311)
-        assert string_world == "0\xb029'45\" -0\xb029'20\" (world)"
+        assert string_world == "0\xb029'45\" −0\xb029'20\" (world)"
```

Comment:
Here I think using unicode strings as suggested would make both the change itself and hypothetical test failures clearer, so I'll change it.